web/cobrands/eastsussex/assets.js

```diff
@@ -0,0 +1,98 @@
+(function(){
+
+if (!fixmystreet.maps) {
+    return;
+}
+
+
+OpenLayers.Format.EastSussex = OpenLayers.Class(OpenLayers.Format.JSON, {
+    read: function(json, type, filter) {
+        if (typeof json == "string") {
+            obj = OpenLayers.Format.JSON.prototype.read.apply(this,
+                                                              [json, filter]);
+        } else {
+            obj = json;
+        }
+
+      var results = [];
+      for (var i=0, len=obj.length; i<len; i++) {
```

Comment:
Could use `$.each` here, but looking at the existing JS it seems that loops are done in both ways in lots of places. So not hugely important 🙂